### ISSUE TITLE
Auto generate passwords

### ISSUE THREAD

**Issue by naxhh** [phansible/phansible#234]
I think we should use some library to generate better passwords.

Default ones are ok, but we should encourage more strong non-predictible passwords...

This is an enhancement and with low priority. But PR's are welcome. As always

**Comment by naxhh** [comment]
This is a nice to have feature.
A quick google search shows to libraries to consider:
https://github.com/ircmaxell/PHP-PasswordLib
https://github.com/hackzilla/password-generator

The task is simple. Generate on phansible random passwords where we are already hard-coding dummy things.
I'll like this to be a interface of a PasswordGenerator that has a public generate method.
And from there inject a specific adapter of one of our implementations.

We can start with any lib and from there choose a better one or just stop there.

**Comment by debo** [comment]
@naxhh what password would you like to generate exactly? Is this just for auto-suggestion during setup or actual provisioning? Let's not forget that this for local development. Could you please elaborte further about the actual use case?

**Comment by naxhh** [comment]
Yep. Ansible is for dev in first instance. But I don't feel ok letting passwords like: 123 as defaults.
Major reason is that people tend to keep those when the project works on Dev and move on to Prod.

It's for phansible UI only. Should not be related with roles. Roles can have a default one or whatever. But phansible should generate a random string. Nothing too complicated neither.

Anyway, this is just an idea, can be rejected anytime

**Comment by debo** [comment]
Nah, I don't want to reject something related to security, I simply wanted to understand better the context and the requirement.
